### DISCUSSION TITLE
[New Feature] Building rubble graphic using a custom palette.

### DISCUSSION THREAD

**Discussion by MikillRosen** [Phobos-developers/Phobos#995] [Ideas and feedback]
### Description:
For some reason, Building rubble graphics use the `iso` palettes which, in my opinion, is inefficient because you would have to copy the building SHP in all theater MIXes (You could make mistakes in this process like Westwood Studios) for the rubble to look right in all theaters when using a palette like `unit` could've lessened the copying and allow the rubble to be used in General MIX files like `genermd.mix` and `isogenmd.mix`

### Possible Implementation:

In artmd.ini
```ini
[SomeBuildingType]
RubblePalette=unit; A palette's file name without the theater suffix and file extension.
```

**Comment by secsome** [comment]
I suppose this feature requires implementation of `PalettesManager`
Releated Address: 47F629
Just hack this palette is enough
